src/Tests/ExecutionPathTracerTests.cs

```diff
@@ -33,7 +32,11 @@ public ExecutionPath GetExecutionPath(string name, int depth = 1)
 
             return tracer.GetExecutionPath();
         }
+    }
 
+    [TestClass]
+    public class IntrisincTests : ExecutionPathTracerTests
```

Comment:
```suggestion
    public class IntrinsicTests : ExecutionPathTracerTests
```

Reply:
I should write a script to catch all my spelling mistakes haha

Reply:
Actually: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker